doc/user_guide/user_guide.md

```diff
@@ -0,0 +1,162 @@
+# Advanced Analytics Framework User Guide
+
+The Advanced Analytics Framework (AAF) provides a general framework to implement complex data analysis algorithms with Exasol. Users can use the features of AAF in their custom implementations.
+
+## Table of Contents
+
+* [Getting Started](#getting-started)
+* [Setup](#setup)
+* [Usage](#usage)
+* [Implementation of Algorithms](#implementation-of-algorithms)
+
+## Getting Started
+
+* Exasol database
+  * The Exasol cluster must already be running with version 7.1 or later.
+  * Database connection information and credentials are needed.
+* TODO: Connection
+* TODO: Algorithm implementation
+
+## Setup
+
+### The Python Package
+
+#### Download The Python Wheel Package
+
+The latest version of the python package of the framework can be downloaded from the Releases in GitHub Repository (see [the latest release](https://github.com/exasol/advanced-analytics-framework/releases/latest)).
+
+Please download the archive `advanced_analytics_framework.whl`.
+
+#### Install The Python Wheel Package
+
+The following command installs the package `advanced-analytics-framework` from [pypi](https://pypi.org):
+
+```bash
+pip install exasol-advanced-analytics-framework
+```
+
+### The Pre-built Script Language Container (SLC)
```

Comment:
I think, we need to try something here, if you look at _Implementation of Algorithms_ , you see a user can configure a UDF, maybe the class can be implemented in the UDF, then a prebuilt container would still make sense.

Reply:
But, maybe it still makes sense, that this framework could be used as a dependency for a separate extension.

Reply:
I think we first need to decide whether AAF should offer a pre-built SLC or not.

If yes, then we probably need to add this pre-built SLC or some logic in order to build it. 
This will probably be addressed in a separate ticket and hence out of scope for the current ticket. 

In any case, I propose to either remove or update the current section in the user guide until the decision and the potentially required changes are applied.

Reply:
if we offer an prebuilt SLC depends on, if my idea works or not. If it works, it makes sense, otherwise not. So, your task is probably trying it out. Built a custom UDF which calls the UDFQueryHandlerRunner with a custom UDFQueryHander.

Reply:
Current experiments showed 
* UDF is in Python module `builtin`
* SLC could be uploaded to BucketFS of database in Docker Container